src/resource/nonRdfData.ts

```diff
@@ -147,17 +153,15 @@ type SaveFileOptions = GetFileOptions & {
 };
 
 /**
- * Saves a file in a folder at a given URL. The server will return the final
- * filename (which may or may not be the given `slug`), it will return it in
- * the response's Location header.
+ * ```{note} This function is still experimental and subject to change, even in a non-major release.
+ * ```
  *
- * If something went wrong saving the file, the returned Promise will be rejected with an Error.
+ * Saves a file in a folder associated with the given URL. The final filename may or may
+ * not be the given `slug`.
  *
- * Please note that this function is still experimental: its API can change in non-major releases.
- *
- * @param folderUrl The URL of the folder where the new file is saved
- * @param file The file to be written
- * @param options Additional parameters for file creation (e.g. a slug)
+ * @param folderUrl The URL of the folder where the new file is saved.
+ * @param file The file to be written.
+ * @param options Additional parameters for file creation (e.g. a slug).
```

Comment:
Could not get this to work.  I keep getting 401 Unauthenticated even though I passed in the fetch
`saveFileInContainer(
            'https://kay.inrupt.net/test',
            new Blob([textPostFile.text], { type: 'plain/text'}, { fetch: session.fetch })
  );`

Only tried the saveFileInContainer.

Reply:
I'll try this locally, but a possible issue is that the container URI must end with a `/`. If that's actually the problem, normalizing the URI in the function might be a good idea (note to myself).

Reply:
The trailing slash does not seem to be required: https://codesandbox.io/s/inruptsolid-client-js-sandbox-forked-8kw7t?file=/src/index.ts

The only difference is that I am not using solid-client-authn's fetch, so presumably the problem lies there?

Reply:
Actually even authenticated, I can't reproduce the issue: slash or no slash, the file is created. See https://codesandbox.io/s/hungry-sky-ycos2?file=/src/App.js

Reply:
Thanks @Vinnl  and @NSeydoux !  Found out why my stuff was goofy.  I've messed up the ending parenthesis for the Blob  - (ugh - it's like being back with Lisp) - it should have been new Blob( [], {} ), { fetch: ... }) instead of  what I had with new Blob( [], {}, { fetch: } ) ).